src/App.jsx

```diff
@@ -22,6 +23,7 @@ function App() {
       <Route path="/help" element={ <Help /> } />
       <Route path="/settings" element={ <Settings />} />
       <Route path="/articles/:article_id" element={ <SingleArticle /> } />
+      <Route path="/articles/:article_id/comments" element={ <CommentsForAnArticle_List /> } />
```

Comment:
Its up to you, but i'd suggest not having a new page for the comments, and just using your <CommentsForAnArticle_List /> component for the `/parricles/:article_id` page.

its slightly confusing at the moment to navigate to a different page that looks the same, but has the comments at the bottom. 

Maybe one to come back to later on though